### PR TITLE
docs: fix typos

### PR DESCRIPTION
**Pull request by omahs**
docs: fix typos

### CODE REVIEW

File: docs/okp4-cognitarium.md [hunk 1]
```diff
@@ -13,13 +13,13 @@ Instantiate message
 |parameter|description|
 |----------|-----------|
 |`limits`|**[StoreLimitsInput](#storelimitsinput)**. Limitations regarding store usage.|
-|`limits.max_byte_size`|**[Uint128](#uint128)**. The maximum number of bytes the store can contains. The size of a triple is counted as the sum of the size of its subject, predicate and object, including the size of data types and language tags if any. Default to [Uint128::MAX] if not set, which can be considered as no limit.<br />**Default:** `"340282366920938463463374607431768211455"`|
```
Comment (ccamel) [inline docs/okp4-cognitarium.md:16]: See [msg.rs#L272](https://github.com/okp4/contracts/blob/b7bc252e3e493dd7d6218bf7373d4454560febb2/contracts/okp4-cognitarium/src/msg.rs#L272) as the description for the `limits.max_byte_size` attribute.

**Review by ccamel — COMMENTED** [review]
Many thanks for the improvements! :heart: 

Please note that markdown files are generated from the source code, including its comments. To ensure persistence, modifications should be made within the comments, not the generated markdown.

It would be so great if you could directly update the comments with the fixes and then regenerate the documentation. Let us know if you need assistance.

**Review by antho31 — APPROVED** [review]

File: contracts/okp4-objectarium/src/state.rs [hunk 2]
```diff
@@ -208,7 +208,7 @@ impl From<BucketLimits> for msg::BucketLimits {
 /// Pagination is the type carrying configuration for paginated queries.
 #[derive(Serialize, Deserialize, Clone, Debug, PartialEq, Eq, JsonSchema)]
 pub struct Pagination {
-    /// The maximum elements a page can contains.
+    /// The maximum elements a page can contain.
```
Comment (amimart) [inline contracts/okp4-objectarium/src/state.rs:211]: It seems some changes on the `objectarium` are not present in the generated docs, maybe you can run the generation script to ensure it comply:

```bash
cargo make schema
cargo make docs-generate
```

**Review by amimart — CHANGES_REQUESTED** [review]
Thank you for pointing that out this is really helpful 🙏 

I just noticed some differences between sources and docs and suggests to run the generation script, see my comment below :)

**Review by ccamel — APPROVED** [review]
Great! Thanks so much :heart:

File: contracts/okp4-objectarium/src/state.rs (hunk 2 above)
Comment (omahs) [reply contracts/okp4-objectarium/src/state.rs:211]: @amimart I'm getting some errors with the generation, just edited the file to match, I hope that's ok. (I'm quite a noob 🙂)

**Review by bdeneux — APPROVED** [review]
Many thanks for this 🙏

**Pull request merged**